Culbert_Exercise10.R

```diff
@@ -0,0 +1,61 @@
+##Question 1: 
+##create plot with MSU vs UW cumulative points scored during a game
+
+#install ggplot
+library(ggplot2)
+
+#view data
+UW_MSU <- read.table("UWvMSU_1-22-13.txt", header = TRUE)
+UW_MSU
+
+#variables for inputting scores
+UW_score = numeric(length(c(1:nrow(UW_MSU))))
+MSU_score = numeric(length(c(1:nrow(UW_MSU))))
+
+#for loop with if-else statements
+for (i in 1:nrow(UW_MSU)){
+  if (UW_MSU$team[i] == "UW"){
+    UW_score[i] <- UW_MSU$score[i]
+    MSU_score[i] <- 0 
+  }else if (UW_MSU$team[i] == "MSU"){
+    MSU_score[i] <- UW_MSU$score[i]
+    UW_score[i] <- 0
+  }
+}
+
+#cumulative sums
+UW_MSU$UW_Total <- c(cumsum(UW_score))
+UW_MSU$MSU_Total <- c(cumsum(MSU_score))
+

```

Comment:
You can do the cumulative sum within the for loop as well:
# load table of scoring
scoring=read.table("UWvMSU_1-22-13.txt",header=TRUE,sep="\t",stringsAsFactors=FALSE)

# look at data
dim(scoring)
head(scoring)

# preallocating matrix to store cumulative scores
cum_scores=matrix(NA,nrow(scoring)+1,3)
cum_scores[,1]=c(0,scoring[,1])
cum_scores[1,2:3]=0
colnames(cum_scores)=c("time","UW","MSU")

# looping through individual scoring events
for(i in 1:nrow(scoring)){
  if(scoring[i,2]=="UW"){
    cum_scores[(i+1),2]=cum_scores[i,2]+scoring[i,3]
    cum_scores[(i+1),3]=cum_scores[i,3]
  }else{
    cum_scores[(i+1),2]=cum_scores[i,2]
    cum_scores[(i+1),3]=cum_scores[i,3]+scoring[i,3]
  }
}